packages/manager-components/src/components/navigation/buttons/translations/translation.ts

```diff
@@ -0,0 +1,27 @@
+import i18next from 'i18next';
+
+import de_DE from './Messages_de_DE.json';
+import en_GB from './Messages_en_GB.json';
+import es_ES from './Messages_es_ES.json';
+import fr_CA from './Messages_fr_CA.json';
+import fr_FR from './Messages_fr_FR.json';
+import it_IT from './Messages_it_IT.json';
+import pl_PL from './Messages_pl_PL.json';
+import pt_PT from './Messages_pt_PT.json';
+
+function addTranslations() {
+  i18next.addResources('de_DE', 'buttons', de_DE);
```

Comment:
we should put the languages in an object to avoid some duplication :
```
const scope = 'buttons';

Object.entries({
  en_GB, de_DE, es_ES, ...
}).forEach(([lng, translations]) => i18next.addResources(lng, scope, translations));
```

Reply:
I fix it 👍